### ISSUE TITLE
The semantics of ./ on Windows compared to Linux

### ISSUE THREAD

**Issue by S-NA** [Microsoft/vscode#45083]
- VSCode Version: Code 1.19.2 (490ef761b76b3f3b3832eff7a588aac891e5fe80, 2018-01-10T15:57:21.071Z)
- OS Version: Linux x64 4.9.84
- Extensions: none
---

Steps to Reproduce:
```bash
mkdir example-dir
cd example-dir
```
1. cat > jsconfig.json
```json
{
    "compilerOptions": {
        "target": "ES6"
        , "module": "commonjs"
    },
    "include": [
        "Declarations/dist/example.d.ts"
        , "./*.js"
    ]
}
^D
```
Open the folder, and notice no error (assuming "Declarations/dist/example.d.ts" actually exists).

Now compress the folder `7za a example-dir.7z example-dir`.
Send the folder to a person on a Windows machine, have them extract it and open the folder. They have, VSCode Version 1.20.1 (f88bbf9137d24d36d968ea6b2911786bfe103002)

Notice the `Unable to open` error. The behaviour of `"Declarations/dist/example.d.ts"` is not similar to *nix. It attempts to include a file from a different path entirely.

In short:
`./` is mandatory on Windows but not on Linux is there a reason why the semantics have to be different? This was a head scratching moment and would be nice if the behaviour was the same regardless of the OS.

Does this issue occur when all extensions are disabled?: Yes
